presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraMetadata.java

```diff
@@ -304,7 +306,7 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
 
         // encode column ordering in the cassandra table comment field since there is no better place to store this
         String columnMetadata = extraColumnMetadataCodec.toJson(columnExtra.build());
-        queryBuilder.append("WITH comment='").append(CassandraSession.PRESTO_COMMENT_METADATA).append(" ").append(columnMetadata).append("'");
+        queryBuilder.append("WITH comment='").append(PRESTO_COMMENT_METADATA).append(" ").append(columnMetadata).append("'");
```

Comment:
I'd inline and remove the constant:
```java
queryBuilder.append("WITH comment='Presto Metadata: ").append(columnMetadata).append("'");

Reply:
it was mechanical.. but will do anyway

Reply:
Actually -- the constant is used in 2 classes (it's just defined twice). So I won't inline, but I will remove one of the declarations.